plugins/plugins.md

```diff
@@ -0,0 +1,14 @@
+<h1 align="center">
+  <img src="../images/logo.png" alt="in-context" width="75" height="75">
+</h1>
+
+# in-context plugins
+
+In-context plugins are a streamlined way to extend in context functionality. 
+
+Following features are available for a plugin to use:
+
+- [client scripts](./client-scripts-api.md): Run scripts with powerful hooks to hook into and change in context browsing behavior. This can be anything from simple debugging tools to advanced emotion and/or eye tracking. 
```

Comment:
hooks - hook sounds weird. And "in context" in that sentence also looks confusing. We should make it capital to make clear it's the name and/or put quotes around it

```suggestion
- [client scripts](./client-scripts-api.md): Run scripts with powerful hooks, which can change "In Context" browsing behavior. This can be anything from simple debugging tools to advanced emotion and/or eye tracking. 
```